src/Math-Polynomials/PMPolynomial.class.st

```diff
@@ -239,20 +239,17 @@ PMPolynomial >> roots [
 { #category : #information }
 PMPolynomial >> roots: aNumber [
 
-	| pol roots root rootFinder |
+	| polynomial roots root rootFinder |
 	rootFinder := PMNewtonZeroFinder with: aNumber.
```

Comment:
Is `aNumber` the starting point for computing the root i.e. 
$`x_0`$

Reply:
Is that a question?

Reply:
Yes. It is a question.

Reply:
aNumber is the initial guess for the root (in PMNewtonZeroFinder x0 is the initial guess for the root, which is passed to the `initValue:` method, which sets the initial value for the iterations)